Library/Homebrew/cli/parser.rb

```diff
@@ -201,32 +224,22 @@ def switch(*names, description: nil, replacement: nil, env: nil, depends_on: nil
       end
       alias switch_option switch
 
-      def value_for_env(env)
-        return if env.blank?
-
-        method_name = :"#{env}?"
-        if Homebrew::EnvConfig.respond_to?(method_name)
-          Homebrew::EnvConfig.public_send(method_name)
-        else
-          ENV.fetch("HOMEBREW_#{env.upcase}", nil)
-        end
-      end
-      private :value_for_env
-
+      sig { params(text: T.nilable(String)).returns(T.nilable(String)) }
       def description(text = nil)
         return @description if text.blank?
```

Comment:
Doesn't need changed now but: this type of method API kinda sucks with Sorbet to me. Better to fall back to getters/setters.

Similarly, I wonder about returning empty strings more often rather than nilable strings.